### DISCUSSION TITLE
when I connect with metamask, after that I want to get account avatar but it comes undefined

### DISCUSSION THREAD

**Discussion by haykoaghababyan** [rainbow-me/rainbowkit#2123] [Q&A]
<ConnectButton.Custom>
      {({
        account,
        chain,
        openAccountModal,
        openChainModal,
        openConnectModal,
        authenticationStatus,
        mounted,
      }) => {}
      
      
      
      account.ensAvatar  is undefined, but in my metamask  I have this avatar
      
      
![image](https://github.com/user-attachments/assets/657994da-6978-4e57-b837-0427c3ad3ce8)

is there any chance to get that avatar?

**Comment by 0xmkh** [comment]
@haykoaghababyan The MetaMask avatar is not the same as ENS avatar. Does your wallet have ENS avatar ? If not you'll not be able to get it.
